code/datums/trading/ai.dm

```diff
@@ -27,24 +27,66 @@ They sell generic supplies and ask for generic supplies.
 	)
 	possible_wanted_items = list(
 		/obj/item/device/                        = TRADER_SUBTYPES_ONLY,
+		/obj/item/device/kit					 = TRADER_BLACKLIST_ALL, // They're impossible to make or get outside of mining, just annoys traders.
 		/obj/item/device/assembly                = TRADER_BLACKLIST_ALL,
 		/obj/item/device/assembly_holder         = TRADER_BLACKLIST_ALL,
-		/obj/item/device/encryptionkey/syndicate = TRADER_BLACKLIST,
+		/obj/item/device/encryptionkey			 = TRADER_BLACKLIST_ALL, //Why does ai want NT encryption keys?
 		/obj/item/device/radio                   = TRADER_BLACKLIST_ALL,
 		/obj/item/device/pda                     = TRADER_BLACKLIST_SUB,
-		/obj/item/device/uplink                  = TRADER_BLACKLIST
+		/obj/item/device/chameleon				 = TRADER_BLACKLIST, // Why should it want a chameleon projector
+		/obj/item/device/dociler				 = TRADER_BLACKLIST, //Item unobtaineable
+		/obj/item/device/flashlight/drone		 = TRADER_BLACKLIST, // No drone stuff
+		/obj/item/device/camera_bug				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/multitool				 = TRADER_BLACKLIST_SUB, // Hacktool, uplink and robo tool ban
+		/obj/item/device/modkit					 = TRADER_BLACKLIST_ALL, // No to modkits
+		/obj/item/device/pin_extractor			 = TRADER_BLACKLIST, // RD's tech
+		/obj/item/device/powersink				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/slime_scanner			 = TRADER_BLACKLIST, //If it was doing slime stuff, it already had this
+		/obj/item/device/spy_bug				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/spy_monitor			 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/suit_cooling_unit/improved = TRADER_BLACKLIST, // Not on Aurora
+		/obj/item/device/taperecorder/cciaa		 = TRADER_BLACKLIST, // Admin item
+		/obj/item/device/batterer				 = TRADER_BLACKLIST, // Item too rare
+		/obj/item/device/contract_uplink		 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/uplink                  = TRADER_BLACKLIST_ALL, // Traitor stuff
+		/obj/item/device/announcer				 = TRADER_BLACKLIST, // Rev item
+		/obj/item/device/ninja_uplink			 = TRADER_BLACKLIST, // Ninja item
+		/obj/item/device/onetankbomb			 = TRADER_BLACKLIST, // Not weapons trader
+		/obj/item/device/kinetic_analyzer		 = TRADER_BLACKLIST, // Not KA trader
+		/obj/item/device/camera					 = TRADER_BLACKLIST_SUB, // a lot of ai/drone/cyborg/fluff items
+		/obj/item/device/uv_light				 = TRADER_BLACKLIST, // CSI item
+		/obj/item/device/nanoquikpay			 = TRADER_BLACKLIST, // eftpos exists
```

Comment:
Not sure what you mean by "eftpos exists"

Reply:
NanoQuikPay is an unused item in the code that is currently unused in game. It functions very similarly to EFTPOS scanner but it's completely different item that the merchant would never be able to find on station. 

[In screenshot, captain is standing above EFTPOS - the other line of items is NanoQuikPay]
![image](https://user-images.githubusercontent.com/6859396/62105980-472ca680-b2a4-11e9-8c83-c991b0c3a127.png)

Now that I've actually spawned it we might want to ban both from traders because under the id the game expects the scanner, instruction booklet AND a wrapped labeled package which spawns code to the machine upon unwrapping. This is very unintuitive.

Reply:
> Now that I've actually spawned it we might want to ban both from traders because under the id the game expects the scanner, instruction booklet AND a wrapped labeled package which spawns code to the machine upon unwrapping. 

I dont think that is correct, as the actual items exist under `/obj/item/device/eftpos` and `/obj/item/device/nanoquikpay` and only during spawn the paper and the box is created, but that doesnt change the path of the item itself and it should still be sellable.

That said, I dont have an issue with blacklisting both because a merchant would have a hard time acquiring either one of those.

Reply:
It will be done